gateway-server/src/main/java/org/apache/knox/gateway/services/security/impl/DefaultKeystoreService.java

```diff
@@ -238,18 +250,15 @@ public boolean isCredentialStoreForClusterAvailable(String clusterName) throws K
 
   @Override
   public boolean isKeystoreForGatewayAvailable() throws KeystoreServiceException {
-    boolean rc;
-    final File  keyStoreFile = new File( config.getIdentityKeystorePath() );
+    final Path keyStoreFilePath = Paths.get(config.getIdentityKeystorePath());
     readLock.lock();
     try {
       try {
```

Comment:
Can we have only one `try-catch-finally` block?

Reply:
Yup.. that was silly.